src/plots/pumps.jl

```diff
@@ -0,0 +1,75 @@
+"""
+Plot the normalized pump curves for the pumps in a network. Displays the plot on the screen
+by default. Use keywords `screen` and `savepath` to control whether to display and/or save
+to file.
+"""
+function plot_pumps(pumps::Dict{String,Any}; screen=true, savepath=nothing)
+    m = 50
+    Qhat = Array(range(0.0, 2.0, length=m))
+    etahat = -Qhat.^2 .+ 2*Qhat
+    Ghat = -1/3*Qhat.^2 .+ 4/3
+    Phat = 1/3*Qhat .+ 2/3
+
+    p1 = Plots.plot()
+    p2 = Plots.plot()
+    p3 = Plots.plot()
+    plotargs = Dict(:lt=>:scatter, :shape=>:circle, :ms=>6, :msc=>:auto)
+    for (key,pump) in pumps
+        Qbep = pump["q_bep"]
+        Gbep = pump["g_bep"]
+        Pbep = pump["P_bep"] # inconsistent capitilization...
+        etabep = 1000.0 * 9.80665 * inv(Pbep) * Gbep * Qbep
```

Comment:
I'd use _WM._DENSITY * _WM._GRAVITY here for the constant terms.

Reply:
Good catch.